sass/mixins/status/status.scss

```diff
@@ -0,0 +1,32 @@
+@import "palette.scss";
+
+@mixin status($type) {
+  display: inline-block;
+  padding: 7px 10px;
+  border-radius: 25px;
+  text-align: center;
+
+  &.alt {
```

Comment:
It just flips the color. I was wondering what would be better:

- add a class on the element
- or add a custom attribute

I went with class as this is a more standard interface. What do you think?